workflows/transcribe-file.go

```diff
@@ -36,14 +36,20 @@ func TranscribeFile(
 	file := paths.MustParse(params.File)
 	destination := paths.MustParse(params.DestinationPath)
 
-	wavFile, err := wfutils.Execute(ctx, activities.Audio.PrepareForTranscriptoion, common.AudioInput{
+	wavFile, err := wfutils.Execute(ctx, activities.Audio.PrepareForTranscription, common.AudioInput{
 		Path:            file,
 		DestinationPath: tempFolder,
 	}).Result(ctx)
 	if err != nil {
 		return err
 	}
 
+	if wavFile == nil {
+		// No error and no result means there was no audio to be processed,
```

Comment:
This feels a bit weird with the comment and all.
Suggestions

- Could we return an error NoAudio? Or would that cause the activity to be considered a failure in temporal?
- Or just remove the comment.

Reply:
> Could we return an error NoAudio? Or would that cause the activity to be considered a failure in temporal?
Yes, that's the case. And it will be retried etc.

> Or just remove the comment.
I commented because it felt a bit weird. Another alternative would be to pull the check out and do it before...